CONTRIBUTING.md

```diff
@@ -18,6 +18,8 @@
     -   Write meaningful commit messages and include the number (`#`) of the issue being resolved (if any) at the end of the commit message.
 
         Example: `:bug: fix: Resolve 'isCorrect' function error (#0)`
+    
+    -   Format your code with [Prettier](https://prettier.io/), and set "printWidth" to 80.
```

Comment:
Can you add the command as well to run it?

Reply:
- Also, I don't think mentioning the 'print width' is necessary, because if the extension is installed in let's say VS Code, the `.prettierrc` file in the project will automatically override the user's local settings.
- The trailing `/` in the URL is unnecessary and should be removed IMO.

Reply:
How I had referred to Prettier in previous projects: [github.com/HarshKapadia2/git_internals/blob/main/CONTRIBUTING.md](https://github.com/HarshKapadia2/git_internals/blob/main/CONTRIBUTING.md#:~:text=Prettier%20should%20be%20used%20to%20format%20code.)